seed-db.sh

```diff
@@ -0,0 +1,21 @@
+# This script rebuilds your local database with
+# a copy of OpenBeta staging database.
+#
+#!/bin/bash
+
+FILE_NAME="openbeta-stg-db.tar.gz"
+REMOTE_FILE="https://storage.googleapis.com/openbeta-dev-dbs/$FILE_NAME"
+
+echo "Downloading db file(s)..."
+#wget --content-disposition $REMOTE_FILE
```

Comment:
@vnugent did you mean to comment this line out? `yarn seed-db` was failing since `openbeta-stg-db.tar.gz` wasn't actually getting downloaded.

Reply:
Ooops, good catch.  Please uncomment it.  Better yet, if you can improve the script so that we can either: download and rebuild db, or rebuild only (skip the download because it's a large file)

Reply:
Uncommenting for now.

For improving the script:
Should the decision to download or skip download be from user input? 

We can automatically start a download if we detect the tar file is outdated. Can we perform any versioning or date comparison between what's on Google storage and what we have locally?